http/Cargo.toml

```diff
@@ -10,6 +10,7 @@ prost-build = "0.6.1"
 vergen = "3.1.0"
 
 [dependencies]
+async-std = { default-features = false, features = ["attributes"], version = "1.6" }
```

Comment:
No additional here dependency because `async-std` is a transient dependency of `ipfs`